src/index.js

```diff
@@ -91,7 +91,7 @@ export default class CGUs extends events.EventEmitter {
 
       console.log(`${logPrefix} Recorded snapshot with id ${snapshotId}.`);
 
-      const document = await filter(pageContent, contentSelector, filters, this._serviceDeclarations[serviceId].filters);
+      const document = await filter(pageContent, contentSelector, removeElements, filters, this._serviceDeclarations[serviceId].filters);
```

Comment:
> I think, at some point it will be necessary to pass the whole `document` object

Reply:
I would be quite in favour of doing that.